pandas/src/klib/khash_python.h

```diff
@@ -2,9 +2,10 @@
 
 #include "khash.h"
 
-// kludge
-
-#define kh_float64_hash_func _Py_HashDouble
+inline khint64_t asint64(double key) {
+  return *(khint64_t *)(&key);
```

Comment:
add a comment (and a link to the python source code) of why we are not using ``_Py_HashDouble`` here (and this issue number as well)